Index.html

```diff
@@ -169,13 +169,17 @@ <h2 class="about">About me</h2>
   </section>
   <section class="page4">
     <p class="pfinal">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-  <form class="cuadrollenado"> 
-    <p class="fill">Full name</p>
-  <img class="fillbar"src="./images/fillbar.svg">
-    <p class="fill">Email address</p> 
-      <img class="fillbar"src="./images/fillbar.svg">
-         <textarea class="comments">Enter text here</textarea>
-</form>
+    
+    <form class="form" method="post"> 
+      <p class="data">Full name</p>
+    <input type="text" name="name" maxlength="30" placeholder="Your name here" required pattern="[A-Za-z0-9]+" required/>
+    <img class="fillbar"src="./images/fillbar.svg">
+    <p class="data">Email adress</p>
+    <input type="email" name="email"  required pattern=".+@gmail+@yahoo+@hotmail\.com" placeholder="example@gmail.com" required/>
+    <img class="fillbar"src="./images/fillbar.svg">
+    <textarea name="textarea" minlength="500" class="data" >Enter text here</textarea>
+  </form>
```

Comment:
- [ ] As stated in the project requirement to collect the data Form should uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html) kindly add `action` Just Like below: 

- `action` value should be similar to https://formspree.io/f/{form_id}

Reply:
- [ ] Kindly follow this example 
<img width="904" alt="Screenshot 2022-11-01 at 15 30 38" src="https://user-images.githubusercontent.com/81444338/199364871-fda32310-2de8-47e2-a89a-11a049d0a292.png">